README.md

```diff
@@ -146,33 +142,126 @@ There are two main steps in the pre-processing pipeline: creating the graph and
 
 The amount of pre-processing required will depend on what kind of datastore you will be using for training.
 
-### Additional inputs
-
-#### MultiZarr Datastore
+### Config
+When creating the graph for the model and for training, a configuration file is needed. The configuration file is a `.yaml` file that contains two dictionaries. One for `datastore` and one for `training`.
+An example of a configuration file for the DANRA dataset is given below:
+```yaml
+datastore:
+  kind: mdp
+  config_path: /path/to/mllam-data-prep-configuration.yaml
+training:
+  state_feature_weights:
+    pres_seasurface: 1.0
+    t2m: 1.0
+    r2m: 1.0
+    u10m: 1.0
+    v10m: 1.0
+```
+A reference to the configuration file used for mllam-data-prep is needed in the `datastore` dictionary. Please see the [mllam-data-prep](https://github.com/mllam/mllam-data-prep) repository for more information on how to create this configuration file.
+The `kind` key in the `datastore` dictionary specifies the type of datastore that will be used (should be one of `npyfiles`, or `mdp`).
 
-* `python -m neural_lam.create_boundary_mask`
-* `python -m neural_lam.create_datetime_forcings`
-* `python -m neural_lam.create_norm`
+### Additional inputs
 
 #### NpyFiles Datastore
 
 #### MDP (mllam-data-prep) Datastore
```

Comment:
How should these two sections be distinguished? The mermaid diagram doesn't only fit to the mllam-data-prep Datastore I guess. I am unsure what to write under "NpyFiles Datastore" if anything?